.circleci/config.yml

```diff
@@ -1,7 +1,6 @@
 # These environment variables must be set in CircleCI UI
 #
 # DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
-# DOCKER_EMAIL   - login info for docker hub
 # DOCKER_USER
```

Comment:
Since you removed the docker hub comment, it's no longer obvious what the other variables are. I suggest adding it back like this:

```
# DOCKER_USER - login user for docker hub
# DOCKER_PASS - login password for docker hub
```

Reply:
Makes sense, kinda figured this was implied though.